spec/features/user_can_delete_a_post_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.feature "Delete a post", type: :feature do
+  scenario "when you click on delete, post is removed from your page" do
+    sign_up
+    click_link "View Your Posts"
+    create_a_post("I love monsieur Travis")
```

Comment:
I suspect that this is not entirely true

Reply:
we will never ever truly know